server/session.go

```diff
@@ -217,9 +215,7 @@ func (p *Plugin) removeUserSession(state *channelState, userID, connID, channelI
 
 	delete(state.Call.Sessions, connID)
 
-	if state.Call.HostID == userID && len(state.Call.Sessions) > 0 {
-		state.Call.HostID = state.Call.getHostID(p.getBotID())
-	}
```

Comment:
A little unsure on why this was removed. If the current host is still in the call why bother with recalculating everything and loop through sessions?

Reply:
If I remember, that was in response to your comment that we should move all the logic into the `getHostID`...
This happens so rarely (comparatively) that I liked the idea of keeping the logic in one place, but your call.

Reply:
Ok, as you like.